nengo_gui/tests/test_data_to_csv.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver import ActionChains
+import time
+import pytest
+from nengo_gui import conftest
+from nengo_gui import testing_tools as tt
+import os
+
+def test_data_to_csv(driver):
+	tt.reset_page(driver)
+
+	tt.update_editor(driver,'''
+import nengo
```

Comment:
Hmm..  except this code requires \n line endings, which would have to be put in manually with Brent's approach, so I'm not sure it makes sense here....  anyone else have thoughts on this?